updateDB.js

```diff
@@ -0,0 +1,88 @@
+var path = require('path')
+var fs = require('fs')
+var findUp = require('find-up')
+var execSync = require('child_process').execSync
+
+var BrowserslistError = require('./error')
+
+function updateDB () {
+  function getLastVersion () {
+    var output = execSync('npm show caniuse-lite version').toString()
+    var lastVersion = output.split('\n')[0]
+
+    return lastVersion
+  }
+
+  function startsWithTwoSpaces (line) {
+    return /^ {2}/.test(line)
+  }
+
+  var lockfile, packageManager
+
+  var packageJson = findUp.sync('package.json')
+  if (!packageJson) {
+    throw new BrowserslistError('Cannot find package.json')
+  }
+
+  var npmLockfile = path.join(packageJson, '/../package-lock.json')
+  var yarnLockfile = path.join(packageJson, '/../yarn.lock')
+
+  if (fs.existsSync(npmLockfile)) {
+    lockfile = npmLockfile
+    packageManager = 'npm'
+  } else if (fs.existsSync(yarnLockfile)) {
+    lockfile = yarnLockfile
+    packageManager = 'yarn'
+  } else {
+    throw new BrowserslistError(
+      'No lockfile found. Run "yarn install" or "npm install"'
+    )
+  }
+
+  console.log('Fetching caniuse-lite ' + getLastVersion() + '...')
+
+  var parsedContent, newContent
+  var content = fs.readFileSync(lockfile, 'utf8')
+
+  if (packageManager === 'npm') {
+    parsedContent = JSON.parse(content)
+
+    if (parsedContent && parsedContent.dependencies) {
+      delete parsedContent.dependencies['caniuse-lite']
```

Comment:
Do you update all nested `caniuse-lite` versions? It could be many different `caniuse-lite` on different nested levels (if I am right about nested nature of `package-lock.json`)

Reply:
Thank you for code review, i will update this function.

Reply:
Thank you for code review, i will update this function.